models/run.py

```diff
@@ -41,7 +41,11 @@ def buildkite_build_message(self):
             if self.reason == "pull-request"
             else ""
         )
-        return f"{self.benchmarkable.type} {self.benchmarkable.id} reason = {self.reason} {pull_request_info} filters = {self.filters}"
+        return (
+            f"{self.benchmarkable.type} {self.benchmarkable.id} reason = {self.reason} {pull_request_info} "
```

Comment:
This isn't a huge deal, but ` {pull_request_info} ` will add two spaces at the end of the message for non-PRs right?

Reply:
Yes, it will. BK UI displays it without double spaces though:
<img width="1057" alt="Screen Shot 2022-06-08 at 11 41 05 AM" src="https://user-images.githubusercontent.com/5522472/172692594-ea808260-6780-4720-8917-4e314ad45b1a.png">